services/QuillLMS/client/app/bundles/Diagnostic/styles/style.scss

```diff
@@ -200,110 +201,6 @@ textarea.submission {
   }
 }
 
-/** The option list should not be visible by default */
```

Comment:
That's cool that we get to drop all of this styling.

Reply:
we love deleting big blocks of code 🎉